Download Expression Matrix for Scanpy/Download SmartSeq2 Expression Matrix as Input to Scanpy.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>Does this need to be sorted out? or is this a by-product of metadata changes?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCellAtlas/data-consumer-vignettes/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>